.travis.yml

```diff
@@ -1,19 +1,17 @@
 sudo: required
+
 services:
 - docker
-language: node_js
-node_js:
-- node
+
 script:
-- npm run lint
-- npm run snyk-test
-- npm run generate-coverage
-- npm run check-coverage
-- npm run upload-coverage-coveralls
+  - docker build --build-arg NODE_ENV=development -t nhsuk/nearby-api .
+  - docker run -d --name mongo "nhsuk/mongodb-pharmacy:$MONGO_IMAGE_TAG"
+  - docker run -e TRAVIS=true -e TRAVIS_JOB_ID=$TRAVIS_JOB_ID -e TRAVIS_BRANCH=$TRAVIS_BRANCH -e COVERALLS_REPO_TOKEN=$COVERALLS_REPO_TOKEN -e SNYK_TOKEN=$SNYK_TOKEN -e TRAVIS_COMMIT=$TRAVIS_COMMIT --link mongo:mongo nhsuk/nearby-api sh -c "cd /code && npm run git-hook && npm run snyk-test && npm run upload-coverage-coveralls"
```

Comment:
Specific versions of `docker-compose` can be downloaded into the environment if needed: https://docs.travis-ci.com/user/docker/#Using-Docker-Compose